scripts/Thin_dcd.py

```diff
@@ -0,0 +1,50 @@
+import neutromeratio
+from openmmtools.constants import kB
+from simtk import unit
+import numpy as np
+import pickle
+import mdtraj as md
+import matplotlib.pyplot as plt
+import sys
+import torch
+from neutromeratio.parameter_gradients import FreeEnergyCalculator
+
+#######################
+temperature = 300 * unit.kelvin
```

Comment:
could import temperature from https://github.com/choderalab/neutromeratio/blob/master/neutromeratio/constants.py , if needed -- doesn't appear to be used in this script though